### PR TITLE
feat/index-sponsor-cards

### PR DESCRIPTION
**Pull request by josix**
## Types of changes

* **New feature**

### Style 
* Update the layout of sponsor cards in index page
* Update style of heading 2
* Update style of SponsorModal

### Behavior
* Allow closing modal by clicking outside of the lightbox.

## Steps to Test This Pull Request
1. Go to the index page 
2. Scroll down to the Sponsors section
3. Click on cards and check the modal layout.

## Expected behavior

localhost_3000_2021_en-us (2)

image

### CODE REVIEW

File: db.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
                         "intro_en_us": "He runs gigantic business",
                         "intro_zh_hant": "馬誼郎，出生於南非的美國企業家",
                         "website_url": "https://docs.djangoproject.com/en/3.1/ref/models/expressions/",
-                        "logo_url": "https://i.imgur.com/RK2bZI7.jpeg"
+                        "logo_url": "https://storage.googleapis.com/pycontw-static/sponsors/kloudless/Frame_183.png?Expires=1650879478&GoogleAccessId=web-storage%40pycontw-225217.iam.gserviceaccount.com&Signature=L2UCoan8HhMYlrr%2B9tdDgQQLND3l%2BWrQ7JxqmMGGDwUq5KHRry3BKZypQZNjNGFDxgK2DBNjWynNGCDHwsFpV%2FhzFPRg2tWwAxnbxNrL58n7Axm0GKA4N%2BkUyPlIh%2Fc%2BZD9yROhUw3rWBWaZGb8RKnMxtg%2FTVP0fXDPnst5xMJDAnvxFTU1fOupCgRIxhtuajtvGG%2BMRVq36gya3pf3zmCx2%2BLOdwqR9n94MDq3Q1ASR2Thb%2Fa9OA1MJp%2BTly9QcW0%2FhZXYEgfNTwHGX%2FFHvazxrHKzxjuBIRJQGHDprQmrs3MH1eTDxrOUqo07zn63bhW7jQGGPPCbDhNzr9G%2BlRQ%3D%3D"
```
Comment (mattwang44) [inline db.json:40]: I'm ok with replacing it with real sponsor logos, but it's actually a signed URL of the GCS object and will be expired tomorrow. If you really want to use the logo here, I think the alternatives are:
- Generate another signed URL but with no expiration time
- Upload to any other free picture storage platform

Comment (josix) [reply db.json]: Oops, thanks for the reminder. I've rollbacked it to the original one.

File: components/sponsors/SponsorModal.vue [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <template>
     <div v-show="shouldShowModal" class="sponsorModal">
+        <div class="closingArea" @click="close"></div>
```
Comment (mattwang44) [inline components/sponsors/SponsorModal.vue:3]: 👍🏽 👍🏽 👍🏽

**Review by mattwang44 — APPROVED** [review]
LGTM Thanks for your brilliant contribution!
I tend to merge this before sending it to the designers for review since some of the changes you made should make others' work easier. If there is any styling issue, I'll get someone else to fix it in another PR.

**Comment by josix** [comment]
That's ok! @anan0202 has reviewed it. I think there is no big issue with the style in the sponsor section.

File: pages/index.vue [hunk 3]
```diff
@@ -42,62 +43,86 @@
...
-                :level-name="leveledSponsors.level_name"
-            >
-                <sponsor-card
-                    v-for="(sponsor, j) in leveledSponsors.sponsors"
-                    :key="`index_sponsor_level_${i}_sponsor_${j}`"
-                    :sponsor-name="sponsor.name_en_us"
-                    :logo-url="sponsor.logo_url"
-                    :tag="getAttributeByLocale(sponsor, 'subtitle')"
-                    @click.native="showModal(sponsor)"
+        <i18n-page-wrapper>
```
Comment (mattwang44) [inline pages/index.vue:50]: I just discovered that the i18nPageWrapper is not imported and registered on this page, so it actually does not take any effects here. @josix Should the wrapper be removed?

Comment (josix) [reply pages/index.vue:50]: Oh, let me fix it. I'll register the `i18nPageWrapper ` and also tune the margin and padding of the content.

**Pull request merged**